bins/revme/src/cmd/evmrunner.rs

```diff
@@ -99,6 +104,25 @@ impl Cmd {
             if self.state {
                 println!("State: {:#?}", out.state);
             }
+            if self.trace {
+                println!("Traces:");
+                let mut evm = Evm::builder()
```

Comment:
You don't need to build a new Evm you can modify the existing one with `evm.modify().with_external_context(...).append_...().build()`